packages/block-library/src/group/edit.native.js

```diff
@@ -29,9 +31,12 @@ function GroupEdit( {
 	}
 
 	return (
-		<InnerBlocks
-			renderAppender={ isSelected && InnerBlocks.ButtonBlockAppender }
-		/>
+		<>
+			<UngroupButton />
```

Comment:
I don't think it's good practice to import a component from `@wordpress/editor` in the block library.

The editor should depend on the block editor which should depend on the block library not the other way around. In general I think we should try to stay as close as possible to the web structure.

In that case they seem to be using the `__experimentalBlockSettingsMenuPluginsExtension` component from `@wordpress/block-editor` to inject the `ConvertToGroupButton` into the `BlockSettingsMenu`.
If that's too much overhead for such a simple PR, we can simply have the `Ungroup` button in the `@wordpress/block-editor` for now and add it directly to the `BlockToolbar`.

Reply:
Ok - thank you for the advice. I have moved src file of button under `@wordpress/block-editor`

However I need this button only for group block so if its ok to render this button as it is currently ?

Reply:
Still, I'd rather we remove that import from `@wordpress/block-editor` inside this block component.
You should be able to test if it's a group component from the BlockToolbar instead

Reply:
Ok, WDYT now ?

Reply:
That looks better thank you :)